### ISSUE TITLE
CIFAR training

### ISSUE THREAD

**Issue by zwzhang123** [DuaneNielsen/oardm#1]
Hi, thanks for your interesting work.

I am struggling to reproducing the DDIM results. Did you have any experience of it?

Training 3k epochs is enough at your code.

[https://github.com/luping-liu/PNDM/issues/6](https://github.com/luping-liu/PNDM/issues/6)

**Comment by RohollahHS** [comment]
> Hi, thanks for your interesting work.
> 
> I am struggling to reproducing the DDIM results. Did you have any experience of it?
> 
> Training 3k epochs is enough at your code.
> 
> [luping-liu/PNDM#6](https://github.com/luping-liu/PNDM/issues/6)

Hi. Did you train this model on CIFAR10 dataset? Can you share the FID and loss curve?

**Comment by DuaneNielsen** [comment]
Here are my  [ results for MNIST](https://wandb.ai/duanenielsen/oardm_binary_mnist?nw=nwuserduanenielsen).  Let me know if you cannot access the project.

**Comment by RohollahHS** [comment]
> Here are my [ results for MNIST](https://wandb.ai/duanenielsen/oardm_binary_mnist?nw=nwuserduanenielsen). Let me know if you cannot access the project.

Thanks. Great.

I trained ARDM on CIFAR10 for about 1000 epochs, but I think the FID is still low compared to the epoch. Do you think the training and the FID value is reasonable?
[results for CIFAR10](https://github.com/user-attachments/files/16110085/run.log)

**Comment by zwzhang123** [comment]
> > Here are my [ results for MNIST](https://wandb.ai/duanenielsen/oardm_binary_mnist?nw=nwuserduanenielsen). Let me know if you cannot access the project.
> 
> Thanks. Great.
> 
> I trained ARDM on CIFAR10 for about 1000 epochs, but I think the FID is still low compared to the epoch. Do you think the training and the FID value is reasonable? [results for CIFAR10](https://github.com/user-attachments/files/16110085/run.log)

I remember my previous repo can obtain the paper's FID, you can have a try.

https://github.com/zzw-zwzhang/pytorch-ddpim

**Comment by RohollahHS** [comment]
> > > Here are my [ results for MNIST](https://wandb.ai/duanenielsen/oardm_binary_mnist?nw=nwuserduanenielsen). Let me know if you cannot access the project.
> > 
> > 
> > Thanks. Great.
> > I trained ARDM on CIFAR10 for about 1000 epochs, but I think the FID is still low compared to the epoch. Do you think the training and the FID value is reasonable? [results for CIFAR10](https://github.com/user-attachments/files/16110085/run.log)
> 
> I remember my previous repo can obtain the paper's FID, you can have a try.
> 
> https://github.com/zzw-zwzhang/pytorch-ddpim

Thanks. Do you have any FID curve during training on CIFAR10?
I want to copmare my model's FID to it to find out how well it is working.
